generator/src/main/java/org/apache/kafka/message/MessageDataGenerator.java

```diff
@@ -2397,6 +2460,9 @@ private String fieldDefault(FieldSpec field) {
                 headerGenerator.addImport(MessageGenerator.BYTES_CLASS);
                 return "Bytes.EMPTY";
             }
+        } else if (field.type().isRecords()) {
+            // TODO should we use some special EmptyRecords class instead?
```

Comment:
Using null as the default seems reasonable to me. We also have `MemoryRecords.EMPTY` which we could use.

Reply:
Ok, going to leave it as `null` since we need to deal with FileRecords and MemoryRecords in different cases.